build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -53,36 +58,52 @@ jobs:
       downloadPath: '$(Build.SourcesDirectory)\Artifacts'
 
   # debugging - remove or comment out before completing PR      
-  # - script: |
-  #     dir /s $(Build.SourcesDirectory)
+  - script: |
+      dir /s $(Build.SourcesDirectory)
 
   - template: ProjectReunion-InstallNuget-Steps.yml
 
-  - ${{if parameters.signConfig }}:
-    # Re-create framework packages because we made them in the build step against unsigned binaries.
-    - template: ProjectReunion-MakeFrameworkPackages-Steps.yml
-
-    # Copy the re-made framework packages into the same location in the artifact drop to be republished at the end of the job.
-    - task: CopyFiles@2
-      inputs:
-        targetFolder: '${{ parameters.nupkgdir }}'
-        sourceFolder: '$(Build.SourcesDirectory)\Artifacts\drop'
-        Contents: '**\Microsoft.ProjectReunion*.appx'
-
-  - powershell: |
-      $prereleaseTag = "${{ parameters.prereleaseVersionTag }}"
-      if ("${{ parameters.useReleaseTag}}" -eq [bool]::TrueString) { $prereleaseTag = "" }
+  # UNDONE to move MRTCore into Project Reunion nuget package
+  # - ${{if parameters.signConfig }}:
+  #   # Re-create framework packages because we made them in the build step against unsigned binaries.
+  #   - template: ProjectReunion-MakeFrameworkPackages-Steps.yml
+
+  #   # Copy the re-made framework packages into the same location in the artifact drop to be republished at the end of the job.
+  #   - task: CopyFiles@2
+  #     inputs:
+  #       targetFolder: '${{ parameters.nupkgdir }}'
+  #       sourceFolder: '$(Build.SourcesDirectory)\Artifacts\drop'
+  #       Contents: '**\Microsoft.ProjectReunion*.appx'
+
+  # - powershell: |
+  #     $prereleaseTag = "${{ parameters.prereleaseVersionTag }}"
+  #     if ("${{ parameters.useReleaseTag}}" -eq [bool]::TrueString) { $prereleaseTag = "" }
+
+  #     & "$(Build.SourcesDirectory)\build\NuSpecs\build-nupkg.ps1" `
+  #       -BuildOutput '$(Build.SourcesDirectory)\Artifacts\drop' `
+  #       -OutputDir '${{ parameters.nupkgdir }}' `
+  #       -prereleaseversion "$prereleaseTag" `
+  #       -DateOverride '$(builddate)' `
+  #       -Subversion '$(buildrevision)' `
+  #       -BuildArch ${{ parameters.primaryBuildArch }} `
+  #       -BuildFlavor ${{ parameters.buildFlavor }}
+
+  #   displayName: 'build-nupkg.ps1'
 
-      & "$(Build.SourcesDirectory)\build\NuSpecs\build-nupkg.ps1" `
-        -BuildOutput '$(Build.SourcesDirectory)\Artifacts\drop' `
-        -OutputDir '${{ parameters.nupkgdir }}' `
-        -prereleaseversion "$prereleaseTag" `
-        -DateOverride '$(builddate)' `
-        -Subversion '$(buildrevision)' `
-        -BuildArch ${{ parameters.primaryBuildArch }} `
-        -BuildFlavor ${{ parameters.buildFlavor }}
-
-    displayName: 'build-nupkg.ps1'
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+    inputs:
+      command: 'pack'
+      packagesToPack: '$(Build.SourcesDirectory)\dev\MRTCore\mrt\NuGet\Microsoft.ProjectReunion.nuspec'
```

Comment:
>MRTCore\mrt\NuGet\Microsoft.ProjectReunion.nuspec [](start = 53, length = 49)

this needs to be outside MRTCore

Reply:
Feels a little weird to have 'Microsoft.ProjectReunion.nuspec' in dev\MRTCore.  I suppose this is just a "phase 1" kind of hting.